package.json

```diff
@@ -45,18 +48,19 @@
     "express-validator": "^6.2.0",
     "helmet": "^3.21.2",
     "i18n": "^0.8.4",
+    "knex": "^0.20.1",
     "mini-css-extract-plugin": "^0.8.0",
     "nunjucks": "^3.2.0",
+    "pg": "^7.12.1",
```

Comment:
Unrelated dependency?

Reply:
This was merged in in an earlier PR, I think it's just a version update, it is being used in my PR so no biggie for me.

Reply:
I think most of these package.json changes were just a re-ordering - when using `npm install`, npm organizes depedencies alphabetically, so if you've previously added deps the end of the list, they can get reorganized by another `npm install` command.